textile/objects-features.textile

```diff
@@ -265,14 +318,30 @@ class LiveObject: // RTLO*
   siteTimeserials: Dict<String, String> // RTLO3b, internal
   createOperationIsMerged: Boolean // RTLO3c, internal
   canApplyOperation(ObjectMessage) -> Boolean // RTLO4a, internal
+  subscribe((LiveObjectUpdate) ->) -> LiveObjectSubscription // RTLO4b
+  unsubscribe((LiveObjectUpdate) ->) // RTLO4c
+  unsubscribeAll() // RTLO4d
+
+interface LiveObjectSubscription: // RTLO4b5
+  unsubscribe() // RTLO4b5a
+
+interface LiveObjectUpdate: // RTLO4b4
+  update: Object // RTLO4b4a
+  noop: Boolean // RTLO4b4b, internal
 
 class LiveCounter extends LiveObject: // RTLC*, RTLC1
   value() -> Number // RTLC5
 
+interface LiveCounterUpdate extends LiveObjectUpdate: // RTLC11, RTLC11a
+  update: { amount: Number } // RTLC11b, RTLC11b1
```

Comment:
Why can't it simply be 
```suggestion
  update: Number  // RTLC11b, RTLC11b1
```
This will make it same as `LiveMapUpdate`, where `update` is at top level instead of nesting.

Reply:
In Java or similar languages, we often have to create redundant classes with accessor methods just to retrieve a single value. Having two levels of accessors to access one piece of data feels inefficient. Also, for the `LiveCounter`, it seems unlikely that any further updates will be needed.